### DISCUSSION TITLE
Account with multiple emails

### DISCUSSION THREAD

**Discussion by cjbuckethead** [community/community#23187] [New to GitHub]
<p>Hello,</p>
<p>I added my work email to my personal github account.</p>
<p>I noticed that Github emails(commits and repository comments) that were intended for my work email was sent to my personal email.</p>
<p>Is there a config somewhere where I can customize this? Like configure emails used if the repo belongs to an organisation?</p>
<p>Because my initial intent for linking the email to Github was so that I don’t have to set up a different Github account, but if work emails are being sent to personal email, that’s concerning especially to the company perspective.</p>
<p>Pls advise.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>You can choose any verified email to receive notifications from your organisation.</p>
<p>You’ll find “custom routing” at the bottom of your notification settings:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/settings/notifications" target="_blank" rel="noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:500/500;"><img src="https://user-images.githubusercontent.com/11161530/181075510-c60d71d7-305b-4f00-9fa9-9ff190a48a27.png" class="thumbnail" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png"></div>

<h3><a href="https://github.com/settings/notifications" target="_blank" rel="noopener">Build software better, together</a></h3>

<p><a href="https://github.com/settings/notifications" target="_blank" rel="noopener">ngs/notifications</a></p>

  <p><span class="label1">GitHub is where people build software. More than 56 million people use GitHub to discover, fork, and contribute to over 100 million projects.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by cjbuckethead** [comment]
<p>Sorry for the late response AND Thank you for this!</p>
